packages/controllers/src/controllers/ChainController.ts

```diff
@@ -554,7 +554,11 @@ const controller = {
     return approvedCaipNetworkIds
   },
 
-  getActiveCaipNetwork() {
+  getActiveCaipNetwork(chainNamespace?: ChainNamespace) {
+    if (chainNamespace) {
+      return state.chains.get(chainNamespace)?.networkState?.caipNetwork || undefined
```

Comment:
No need to do `|| undefined` I think since if property doesn't exist it'll return undefined already